modules/claims_api/app/services/claims_api/unsynchronized_evss_claims_service.rb

```diff
@@ -25,6 +25,19 @@ def all
       end.flatten
     end
 
+    def count
```

Comment:
It may be implied, but I think maybe renaming this to `claim_count` might make more sense. Feels weird to have `service.count`  -- personal preference though up to you

Reply:
Sounds good, I'll make the change.

Reply:
Done